infrastructure/local/seed.hlb.yml

```diff
@@ -1,14 +1,14 @@
 self:
```

Comment:
Should we update `seed-example.yml` as well, to include `happy-life-bank` ?

Reply:
Where is `seed-example.yml`?